examples/tera_templates/src/tests.rs

```diff
@@ -49,7 +49,13 @@ fn test_name() {
 
         let expected = Template::show(client.rocket(), "index", &context).unwrap();
         assert_eq!(response.status(), Status::Ok);
-        assert_eq!(response.into_string(), Some(expected));
+
+        let response_str = response.into_string().unwrap_or_default();
+        assert_eq!(response_str, expected);
+        assert!(response_str.contains("Jack"));
+        assert!(response_str.contains("One"));
+        assert!(response_str.contains("Two"));
+        assert!(response_str.contains("Three"));
```

Comment:
This is equivalent to what we had before. Please revert.

Reply:
It's actually a bit more than before. The original code tests if the response contents are the same as the rendered template, but it doesn't verify that the values from the context are displayed in that rendering. You could comment out the whole include line from the template and the tests would still pass.
So I still think it's a useful test, but I agree, it's a bit weird here. The tera repository probably would be better place for a test like this, because it's not really testing any Rocket features. I'll revert it.

Reply:
You're right, there's nothing actually testing that we have those items in there. It's a bit of a weird case because, in reality, these items wouldn't be static, and you'd want to check that the rendered template contains the dynamic items. So it's a bit weird to test them in this case. If we extend this in the future to make the items dynamic, then I think it would make sense to test this. Thanks for your comment.